docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -494,15 +494,15 @@ public class FileConfig {
     String format;
 
     /**
-     * The file log level.
+     * The level of logs to be written into the file.
      */
     @ConfigItem(defaultValue = "ALL")
     Level level;
 
     /**
-     * The file logging log level.
+     * The name of file where logs will be stored.
      */
-    @ConfigItem(defaultValue = "quarkus.log")
+    @ConfigItem(defaultValue = DEFAULT_LOG_FILE_NAME)
```

Comment:
The constant is not defined here.

Reply:
@gsmet I didn't want to confuse reader readers with `quarkus.log` file name vs `quarkus.log` namespace and thus I didn't include it here on purpose. I thought that having a static placeholder will be enough to keep it simple and readable but not confusing. IMO, since this code is just a sample used for a demonstration purpose and to explain how configuration structures and nesting can be done, it does not have to be as strict as the runnable code. Can we leave this as it is? I would like to avoid having any mention of `quarkus.log` in terms other than a namespace, here.

Reply:
We need this example to compile. We can't give a doc example that doesn't compile. So if you don't want to use the constant, just use `quarkus.log` as it was before. It is just an example so it doesn't need to be totally similar to the code we have in Quarkus.

Reply:
Will change to "application.log" then. As I said before - I do not want to have `quarkus.log` file name because it's confusing with `quarkus.log` log namespace. By doing this it will compile and will remain non confusing :)